s-pilot

```diff
@@ -526,6 +538,8 @@ code_chat_function(){
 				response_data=$(echo $response | jq -r '.content[0].text')
 			elif [[ "$USE_API" == "moonshot" ]]; then
 				response_data=$(echo $response | jq -r '.choices[].message.content')
+      elif [[ "$USE_API" == "novita" ]]; then
+        response_data=$(echo $response | jq -r '.choices[].message.content')
 			fi
```

Comment:
indent

Reply:
Should I use "/t" instead of spaces to indent here?